share/instruments/generate_tsv.py

```diff
@@ -12,8 +12,7 @@ def eprint(*args, **kwargs):
     raise SystemExit(1)
 
 import csv
-import os
-import sys
+from pathlib import Path
```

Comment:
I generally prefer `import X` rather than `from X import Y`. This forces you to write `X.Y` in the code rather than just `Y`, but it makes it clear where `Y` came from which might not be immediately obvious otherwise.